### PR TITLE
N7s aur#187 aocs/gnc demo

### PR DESCRIPTION
**Pull request by igorJarek**

### CODE REVIEW

File: requirements/AOCS-GNC_AURORA_0001/taste_project/DataView.aadl [hunk 1]
```diff
@@ -0,0 +1,636 @@
+--------------------------------------------------------
```
Comment (Lurkerpas) [inline requirements/AOCS-GNC_AURORA_0001/taste_project/DataView.aadl]: This file is automatically generated - please remove

File: requirements/AOCS-GNC_AURORA_0001/taste_project/Default_Deployment.aadl [hunk 2]
```diff
@@ -0,0 +1,130 @@
+PACKAGE deploymentview::DV::Node_Linux
```
Comment (Lurkerpas) [inline requirements/AOCS-GNC_AURORA_0001/taste_project/Default_Deployment.aadl]: This file is automatically generated - please remove

File: requirements/AOCS-GNC_AURORA_0001/taste_project/DeploymentView.aadl [hunk 3]
```diff
@@ -0,0 +1,130 @@
+PACKAGE deploymentview::DV::Node_Linux
```
Comment (Lurkerpas) [inline requirements/AOCS-GNC_AURORA_0001/taste_project/DeploymentView.aadl]: This file is automatically generated - please remove

File: requirements/AOCS-GNC_AURORA_0001/taste_project/InterfaceView.aadl [hunk 4]
```diff
@@ -0,0 +1,706 @@
+---------------------------------------------------
```
Comment (Lurkerpas) [inline requirements/AOCS-GNC_AURORA_0001/taste_project/InterfaceView.aadl]: This file is automatically generated - please remove

File: requirements/AOCS-GNC_AURORA_0001/taste_project/QGen_Tutorial.acn [hunk 5]
```diff
@@ -0,0 +1,12 @@
+QGEN-TUTORIAL-DATAVIEW DEFINITIONS ::= BEGIN
+
+   --  In this ACN model, you can specify custom binary encoding rules for
```
Comment (Lurkerpas) [inline requirements/AOCS-GNC_AURORA_0001/taste_project/QGen_Tutorial.acn]: Please remove the commented out code

File: requirements/AOCS-GNC_AURORA_0001/taste_project/QGen_Tutorial.msc [hunk 6]
```diff
@@ -0,0 +1,32 @@
+mscdocument QGen_Tutorial /* MSC AND */;
```
Comment (Lurkerpas) [inline requirements/AOCS-GNC_AURORA_0001/taste_project/QGen_Tutorial.msc]: This file is not needed, please remove

File: requirements/AOCS-GNC_AURORA_0001/taste_project/QGen_Tutorial.pro.user [hunk 7]
```diff
@@ -0,0 +1,253 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (Lurkerpas) [inline requirements/AOCS-GNC_AURORA_0001/taste_project/QGen_Tutorial.pro.user]: This file is not needed in the repository, consider providing .gitignore

File: requirements/AOCS-GNC_AURORA_0001/taste_project/QGen_Tutorial.pro.user.4.9-pre1 [hunk 8]
```diff
@@ -0,0 +1,287 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (Lurkerpas) [inline requirements/AOCS-GNC_AURORA_0001/taste_project/QGen_Tutorial.pro.user.4.9-pre1]: Please delete this file

File: requirements/AOCS-GNC_AURORA_0001/taste_project/rtems.dv.xml [hunk 9]
```diff
@@ -0,0 +1,11 @@
+<DeploymentView>
+    <Node name="GR712RC_RTEMS6_SMP_QDP_1" type="ocarina_processors_leon::leon3.rtems_smp_qdp" node_label="Node_1" namespace="ocarina_processors_leon">
```
Comment (Lurkerpas) [inline requirements/AOCS-GNC_AURORA_0001/taste_project/rtems.dv.xml]: Was this tested on RTEMS? MMI won't work there...

File: requirements/AOCS-GNC_AURORA_0001/taste_project/linux.dv.xml [hunk 10]
```diff
@@ -0,0 +1,26 @@
+<DeploymentView>
+    <Node name="x86 Linux CPP_1" type="ocarina_processors_x86::x86.generic_linux" node_label="Node_1" namespace="ocarina_processors_x86">
```
Comment (Lurkerpas) [inline requirements/AOCS-GNC_AURORA_0001/taste_project/linux.dv.xml]: This DV seems to be incomplete. Where is MMI deployed?

File: requirements/AOCS-GNC_AURORA_0001/taste_project/work/measurer_and_actuator/implem/default/Ada/src/measurer_and_actuator.adb [hunk 11]
```diff
@@ -0,0 +1,57 @@
+------------------------------------------------------------------------------
```
Comment (Lurkerpas) [inline requirements/AOCS-GNC_AURORA_0001/taste_project/work/measurer_and_actuator/implem/default/Ada/src/measurer_and_actuator.adb]: Do we need the ada implementation?

File: requirements/AOCS-GNC_AURORA_0001/taste_project/work/simulated_acs_hw/implem/default/Ada/src/simulated_acs_hw.adb [hunk 12]
```diff
@@ -0,0 +1,121 @@
+------------------------------------------------------------------------------
```
Comment (Lurkerpas) [inline requirements/AOCS-GNC_AURORA_0001/taste_project/work/simulated_acs_hw/implem/default/Ada/src/simulated_acs_hw.adb]: Do we need the ada implementation?

File: requirements/AOCS-GNC_AURORA_0001/taste_project/work/simulated_acs_hw/implem/cfs/C/src/simulated_acs_hw.c [hunk 13]
```diff
@@ -0,0 +1,187 @@
+//  ------------------------------------------------------------------------
+//
+//                           Q G e n   T u t o r i a l
+//
```
Comment (Lurkerpas) [inline requirements/AOCS-GNC_AURORA_0001/taste_project/work/simulated_acs_hw/implem/cfs/C/src/simulated_acs_hw.c]: Is this needed?

File: requirements/AOCS-GNC_AURORA_0001/taste_project/work/simulated_acs_hw/implem/uart_comm/C/src/simulated_acs_hw.c [hunk 14]
```diff
@@ -0,0 +1,213 @@
+//  ------------------------------------------------------------------------
+//
```
Comment (Lurkerpas) [inline requirements/AOCS-GNC_AURORA_0001/taste_project/work/simulated_acs_hw/implem/uart_comm/C/src/simulated_acs_hw.c]: Is this needed?

File: requirements/AOCS-GNC_AURORA_0001/taste_project/DataView.aadl (hunk 1 above)
Comment (igorJarek) [reply requirements/AOCS-GNC_AURORA_0001/taste_project/DataView.aadl]: Done

File: requirements/AOCS-GNC_AURORA_0001/taste_project/Default_Deployment.aadl (hunk 2 above)
Comment (igorJarek) [reply requirements/AOCS-GNC_AURORA_0001/taste_project/Default_Deployment.aadl]: Done

File: requirements/AOCS-GNC_AURORA_0001/taste_project/DeploymentView.aadl (hunk 3 above)
Comment (igorJarek) [reply requirements/AOCS-GNC_AURORA_0001/taste_project/DeploymentView.aadl]: Done

File: requirements/AOCS-GNC_AURORA_0001/taste_project/InterfaceView.aadl (hunk 4 above)
Comment (igorJarek) [reply requirements/AOCS-GNC_AURORA_0001/taste_project/InterfaceView.aadl]: Done

File: requirements/AOCS-GNC_AURORA_0001/taste_project/QGen_Tutorial.acn (hunk 5 above)
Comment (igorJarek) [reply requirements/AOCS-GNC_AURORA_0001/taste_project/QGen_Tutorial.acn]: Done

File: requirements/AOCS-GNC_AURORA_0001/taste_project/QGen_Tutorial.msc (hunk 6 above)
Comment (igorJarek) [reply requirements/AOCS-GNC_AURORA_0001/taste_project/QGen_Tutorial.msc]: Done

File: requirements/AOCS-GNC_AURORA_0001/taste_project/QGen_Tutorial.pro.user (hunk 7 above)
Comment (igorJarek) [reply requirements/AOCS-GNC_AURORA_0001/taste_project/QGen_Tutorial.pro.user]: Done

File: requirements/AOCS-GNC_AURORA_0001/taste_project/QGen_Tutorial.pro.user.4.9-pre1 (hunk 8 above)
Comment (igorJarek) [reply requirements/AOCS-GNC_AURORA_0001/taste_project/QGen_Tutorial.pro.user.4.9-pre1]: Done

File: requirements/AOCS-GNC_AURORA_0001/taste_project/rtems.dv.xml (hunk 9 above)
Comment (igorJarek) [reply requirements/AOCS-GNC_AURORA_0001/taste_project/rtems.dv.xml]: Removed

File: requirements/AOCS-GNC_AURORA_0001/taste_project/linux.dv.xml (hunk 10 above)
Comment (igorJarek) [reply requirements/AOCS-GNC_AURORA_0001/taste_project/linux.dv.xml]: Removed
MMI uploaded

File: requirements/AOCS-GNC_AURORA_0001/taste_project/work/measurer_and_actuator/implem/default/Ada/src/measurer_and_actuator.adb (hunk 11 above)
Comment (igorJarek) [reply requirements/AOCS-GNC_AURORA_0001/taste_project/work/measurer_and_actuator/implem/default/Ada/src/measurer_and_actuator.adb]: No, Removed

File: requirements/AOCS-GNC_AURORA_0001/taste_project/work/simulated_acs_hw/implem/default/Ada/src/simulated_acs_hw.adb (hunk 12 above)
Comment (igorJarek) [reply requirements/AOCS-GNC_AURORA_0001/taste_project/work/simulated_acs_hw/implem/default/Ada/src/simulated_acs_hw.adb]: No, Removed

File: requirements/AOCS-GNC_AURORA_0001/taste_project/work/simulated_acs_hw/implem/cfs/C/src/simulated_acs_hw.c (hunk 13 above)
Comment (igorJarek) [reply requirements/AOCS-GNC_AURORA_0001/taste_project/work/simulated_acs_hw/implem/cfs/C/src/simulated_acs_hw.c]: No, Removed

File: requirements/AOCS-GNC_AURORA_0001/taste_project/work/simulated_acs_hw/implem/uart_comm/C/src/simulated_acs_hw.c (hunk 14 above)
Comment (igorJarek) [reply requirements/AOCS-GNC_AURORA_0001/taste_project/work/simulated_acs_hw/implem/uart_comm/C/src/simulated_acs_hw.c]: No, Removed

**Review by Lurkerpas — APPROVED** [review]

**Pull request merged**